src/hooks.js

```diff
@@ -337,3 +306,52 @@ export function useLocalIdentity(entity) {
 
   return { name }
 }
+
+/* eslint-disable react-hooks/rules-of-hooks */
+export function useArrowKeysFocusRefs(refs) {
```

Comment:
There are two functions here that aim to solve the same problem: cycle through a list using arrow keys.
The first approach is used in the local autocomplete component - this will get removed.
The second approach is the one I submitted to `aragon-ui`, and the one that will live to see another day.

I was planning on removing the first one when the autocomplete component was merged and released to `aragon-ui` but that is taking some time, so for now I added the other version of the hook, to use it with the `ButtonDropDown` component.

When this is merged and the autocomplete component is merged I will come back and do the corresponding cleanup.